tasks/44_time-simulation/time-simulation.js

```diff
@@ -0,0 +1,18 @@
+export class SimTime {
+  constructor() {
+    this.time = 0;
+    this.speed = 1;
+    this.realTime = 0;
+  }
+
+  get() {
+    return this.time;
+  }
+  setSpeed(speed) {
+    this.speed = speed;
+  }
+  update(realTime) {
+    this.time = this.time + (realTime - this.realTime) * this.speed;
```

Comment:
```suggestion
    this.time = this.time + (realTime - this.realTime) * this.speed;
```
```suggestion
const delta = realTime - this.realTime
const deltaAccelerated = delta * this.speed
this.time += deltaAccelerated
```
Don't have to put everything in one line.
Sometimes the code is more clear with some additional and "unnecessary" variables. But this variables give something to the clarity of the code.